horse/recommendation/base.py

```diff
@@ -0,0 +1,3 @@
+class BaseRecommendationService:
```

Comment:
- If that's an interface I suggest we make it an `abc` (metaclass=ABCMeta).
- Again, if that's an interface I suggest we drop the `Base` prefix
- `recommend` should be run for a specific user, so the signature should probably be `recommend(self, user)`